tools/system_libs.py

```diff
@@ -535,6 +535,10 @@ class MuslInternalLibrary(Library):
   ]
 
 
+class WasiLibrary(Library):
+  includes = [['system', 'include', 'wasi']]
```

Comment:
This seems unnecessary  .. why not just "include <wasi/wasi.h>" which is always available?

Reply:
Good idea, switching to `wasi/wasi.h`